prog/postgres/postgres_server_nexus.rb

```diff
@@ -31,14 +31,11 @@ def self.assemble(resource_id:, timeline_id:, timeline_access:)
         enable_ip4: true
       )
 
-      vm_st.subject.private_subnets.each { _1.firewall_rules.each(&:destroy) }
-      vm_st.subject.add_allow_ssh_fw_rules(vm_st.subject.private_subnets.first)
+      vm_st.subject.firewall_rules.map(&:destroy)
+      vm_st.subject.add_allow_ssh_fw_rules
```

Comment:
We have this pattern at 3 different places (pg server, GitHub runner, vm pool). Probably it's needed for all virtual machines with sshable a.k.a service vms.

What do you think about add a flag to `Vm::Nexus.assemble` as `allow_only_ssh_connection` or shorter? So we can set it `true` for vms that provisioned with `assemble_with_sshable`. No need to destroy and add 22 for them.

```ruby
# nexus.rb
# def self.assemble

fw = Firewall.create_with_id(vm_id: vm.id, name: "#{name}-default")

if allow_only_ssh_connection
    ["0.0.0.0/0", "::/0"].each { |ip| fw.insert_firewall_rule(ip, Sequel.pg_range(22..22)) }
else
    ["0.0.0.0/0", "::/0"].each { |ip| fw.insert_firewall_rule(ip, nil) }
end


# def self.assemble_with_sshable(unix_user, *, **kwargs)
kwargs[:allow_only_ssh_connection] = true
```

Reply:
I have updated the code. I used a little bit of a different approach to select the port_range. Thanks for the suggestion.